README.md

```diff
@@ -36,21 +36,21 @@ For more information, you can refer to [ChainerRL's documentation](http://chaine
 
 ## Algorithms
 
-| Algorithm | Discrete Action | Continous Action | Recurrent Model | CPU Async Training |
-|:----------|:---------------:|:----------------:|:---------------:|:------------------:|
-| DQN (including DoubleDQN etc.) | ✓ | ✓ (NAF) | ✓ | x |
-| Categorical DQN | ✓ | x | ✓ | x |
-| Rainbow | ✓ | x | ✓ | x |
-| IQN | ✓ | x | ✓ | x |
-| DDPG | x | ✓ | ✓ | x |
-| A3C  | ✓ | ✓ | ✓ | ✓ |
-| ACER | ✓ | ✓ | ✓ | ✓ |
-| NSQ (N-step Q-learning) | ✓ | ✓ (NAF) | ✓ | ✓ |
-| PCL (Path Consistency Learning) | ✓ | ✓ | ✓ | ✓ |
-| PPO  | ✓ | ✓ | ✓ | x |
-| TRPO | ✓ | ✓ | ✓ | x |
-| TD3 | x | ✓ | x | x |
-| SAC | x | ✓ | x | x |
+| Algorithm | Discrete Action | Continous Action | Recurrent Model | Batch Training | CPU Async Training |
+|:----------|:---------------:|:----------------:|:---------------:|:--------------:|:------------------:|
+| DQN (including DoubleDQN etc.) | ✓ | ✓ (NAF) | ✓ | ✓ | x |
+| Categorical DQN | ✓ | x | ✓ | ✓ | x |
+| Rainbow | ✓ | x | ✓ | ✓ | x |
+| IQN | ✓ | x | ✓ | ✓ | x |
+| DDPG | x | ✓ | ✓ | ✓ | x |
+| A3C  | ✓ | ✓ | ✓ | ✓ (A2C) | ✓ |
+| ACER | ✓ | ✓ | ✓ | x | ✓ |
+| NSQ (N-step Q-learning) | ✓ | ✓ (NAF) | ✓ | ✓ | ✓ |
```

Comment:
x https://github.com/chainer/chainerrl/blob/master/chainerrl/agents/nsq.py

Reply:
Ok, so essentially remove the check mark?